packages/localize/index.android.ts

```diff
@@ -41,6 +41,10 @@ export function androidLaunchEventLocalizationHandler() {
 }
 
 export function overrideLocale(locale: string): boolean {
-	ApplicationSettings.setString('__app__language__', locale.substring(0, 2));
+	let language_code = locale.substring(0, 2)
+	if (locale.indexOf('-') < 0 && locale.length == 3) {
+		language_code = locale
```

Comment:
This looks good. Please add semicolon on lines 44 and 46 for consistency sake as the rest of the file uses it.